### PR TITLE
Add Croatian translation

### PR DESCRIPTION
**Pull request by milotype**
If necessary, please also update `template.dotx` and `template.ott`.
I don't know how to do such things ...

### CODE REVIEW

File: content/hr/blog/sample-letter-gdpr-access-request/index.md [hunk 1]
```diff
@@ -8,36 +8,39 @@
...
-<span class="blog-letter-fill-in">Ime</span></p>
+<p>S poštovanjem</p>
+
+<p><span class="blog-letter-fill-in">Ovdje upiši svoje ime.</span></p>
 </div>
+
+<a href="/downloads/primjerak-pisma-gdpr-zahtjev-pristup-osobnipodaci.org.ott" class="button button-primary" style="margin-bottom: 10px;">Download LibreOffice or OpenOffice template (.ott)</a><br>
+<a href="/downloads/primjerak-pisma-gdpr-zahtjev-pristup-osobnipodaci.org.dotx" class="button button-secondary" style="margin-bottom: 10px;">Download Word template (.dotx)</a><br>
+<a href="/downloads/primjerak-pisma-gdpr-zahtjev-pristup-osobnipodaci.org.pdf" class="button button-secondary" style="margin-bottom: 10px;">Download PDF form (.pdf)</a><br>
+<a href="/downloads/primjerak-pisma-gdpr-zahtjev-pristup-osobnipodaci.org.txt" class="button button-secondary">Download plain text file (.txt)</a>
```
Comment (baltpeter) [inline content/hr/blog/sample-letter-gdpr-access-request/index.md:46]: Can you please provide translations for these strings?

* Download LibreOffice or OpenOffice template (.ott)
* Download Word template (.dotx)
* Download PDF form (.pdf)
* Download plain text file (.txt)

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks, @milotype, and sorry for taking so long with the review!

I have generated the sample letters and linked them in the blog posts. There's a couple of translations that we still need. It'd be great if you could provide those as well.

**Comment by milotype** [comment]
@baltpeter I did the necessary changes ... I hope it's as you imagined :-)

**Comment by milotype** [comment]
@baltpeter I've also noticed that some _Croatian_ filenames in **static/downloads** folder need changes!
I don't know how to change those filenames ... sorry. Below is a list of how those filenames should change. I hope you can handle that, or tell me what to do.

https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/primjerak-pisma-gdpr-zahtjev-pristup-osobnipodaci.org.dotx
into
https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/uzorak-pismo-gdpr-pristup-zahtjev-osobnipodaci.org.dotx

https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/primjerak-pisma-gdpr-zahtjev-pristup-osobnipodaci.org.ott
into
https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/uzorak-pismo-gdpr-pristup-zahtjev-osobnipodaci.org.ott

https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/primjerak-pisma-gdpr-zahtjev-pristup-osobnipodaci.org.pdf
into
https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/uzorak-pismo-gdpr-pristup-zahtjev-osobnipodaci.org.pdf

https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/primjerak-pisma-gdpr-zahtjev-pristup-osobnipodaci.org.txt
into
https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/uzorak-pismo-gdpr-pristup-zahtjev-osobnipodaci.org.txt



https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/sample-letter-gdpr-objection-request-osobnopodaci.org.dotx
into
https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/uzorak-pismo-gdpr-prigovor-zahtjev-osobnopodaci.org.dotx

https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/sample-letter-gdpr-objection-request-osobnopodaci.org.ott
into
https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/uzorak-pismo-gdpr-prigovor-zahtjev-osobnopodaci.org.ott

https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/sample-letter-gdpr-objection-request-osobnopodaci.org.pdf
into
https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/uzorak-pismo-gdpr-prigovor-zahtjev-osobnopodaci.org.pdf

https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/sample-letter-gdpr-objection-request-osobnopodaci.org.txt
into
https://github.com/milotype/website-1/blob/milotype-2021-11-15/static/downloads/uzorak-pismo-gdpr-prigovor-zahtjev-osobnopodaci.org.txt

**Review by baltpeter — APPROVED** [review]
Thank you. :)

**Pull request merged**